content/content/Usage/plugin management.md

```diff
@@ -0,0 +1,160 @@
+#### Example
+
+```
+crane plugin-manager list
+Listing from the repo default
++-----------------+------------------+-------------------+
+|      NAME       | SHORTDESCRIPTION | AVAILABLEVERSIONS |
++-----------------+------------------+-------------------+
+| OpenshiftPlugin | OpenshiftPlugin  | v0.0.1            |
++-----------------+------------------+-------------------+
+
+```
+
+#### Other valid execution examples
+```
+crane plugin-manager --installed     # lists installed plugins
+crane plugin-manager --params -n foo # lists all the version of plugin foo with detailed information.
+```
+
+### Add plugin utility
+
+Add any plugin to a plugin directory to be consumed by transform command.
+
+```
+crane plugin-manager add -h
+installs the desired plugin
+
+Usage:
+  crane plugin-manager add <name> [flags]
+
+Flags:
+  -h, --help             help for add
+      --version string   Install specific plugin version (if not passed, installs latest plugin version or the only available one)
+
+Global Flags:
+  -c, --config-file string   Path to config file which contains a yaml representation of cli flags. Explicit flags take precedence over config file values.
+      --debug                Debug the command by printing more information
+  -p, --plugin-dir string    The path where binary plugins are located (default "plugins")
+      --repo string          The name of the repository from which to list the plugins from
+```
+
+#### Example
+
+```
+crane plugin-manager add foo
+```
+
+The above command downloads the binary of plugin `foo` from appropriate source and places it in plugin directory (default is `plugins`).
+
+### Remove plugin utility
+
+Remove any unwanted plugins that are not needed anymore to exempt them being consumed by transform command.
+
+```
+crane plugin-manager remove -h
+removes the desired plugin
+
+Usage:
+  crane plugin-manager remove <name> [flags]
+
+Flags:
+  -h, --help   help for remove
+
+Global Flags:
+  -c, --config-file string   Path to config file which contains a yaml representation of cli flags. Explicit flags take precedence over config file values.
+      --debug                Debug the command by printing more information
+  -p, --plugin-dir string    The path where binary plugins are located (default "plugins")
+      --repo string          The name of the repository from which to list the plugins from
+```
+
+#### Example
+
+```
+crane plugin-manager remove foo
+```
+
+The above command will remove the plugin `foo` from the plugin directory.
+
+
+### Manual plugin management
+
+Right now there are only two plugins available, there might be more plugins available soon. Currently available plugins are [Kubernetes](https://github.com/konveyor/crane-lib/tree/main/transform/kubernetes) (build into crane-lib) and [OpenShift](https://github.com/konveyor/crane-plugin-openshift). These plugins can be added to desired plugin directory (default is `plugin` directory where `crane` is installed) by following methods -
```

Comment:
Replace "These plugins..." with "With the exception of the kubernetes plugin, which doesn't need to be installed as it's built in, these plugins..."